### PR TITLE
style: Descriptions layout should be fixed when bordered is true

### LINKED ISSUE #46397: Descriptions 使用 borderd 时，会有列挤压问题
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/dai-bian-kuang-de-antd-5-12-1-forked-4t8t26)

### Steps to reproduce

descriptions 分栏，使用 borderd 属性，将某个 item children 填充超长

### What is expected?

期望列宽固定，内容换行

### What is actually happening?

将其他列进行挤压

| Environment | Info |
| --- | --- |
| antd | 5.12.2 |
| React | 18.2.0 |
| System | macos |
| Browser | Chrome Version 120.0.6099.71 |

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/46397

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix Descriptions fixed layout when `bordered` is true.        |
| 🇨🇳 Chinese | 修复 Descriptions 开启 `bordered` 时的内容布局未固定的问题。          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: components/descriptions/style/index.ts [hunk 1]
```diff
@@ -58,7 +58,6 @@ const genBorderedStyle = (token: DescriptionsToken): CSSObject => {
       [`> ${componentCls}-view`]: {
         border: `${unit(token.lineWidth)} ${token.lineType} ${token.colorSplit}`,
         '> table': {
-          tableLayout: 'auto',
```
Comment (afc163) [inline components/descriptions/style/index.ts:61]: 看上去确实没必要。

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/issues/46397#issuecomment-1853236715

**Comment by MadCcc** [comment]
区别是换不换行么

**Comment by afc163** [comment]
See https://github.com/ant-design/ant-design/issues/46397#issuecomment-1853761343

**Pull request closed without merging**